.github/scripts/supported_models/generate_file.py

```diff
@@ -14,121 +14,121 @@
 sys.path.insert(
     1,
     os.path.abspath(
-        os.path.join(Path(__file__), "../../../../custom_components/powercalc"),
```

Comment:
I prefer to have double quotes as this is also used throughout HA core codebase.
PEP8 sais following https://peps.python.org/pep-0008/#string-quotes.
So I think it's mainly a taste issue / personal preference.

Reply:
@LDprg Could you revert this quote change and use the double quotes again?

I'm not sure why some of the github actions are failing yet.
I'll quickly try to clear runner caches. When it's not fixed I'll have a further look when I have some time available.

Reply:
Change reverted the quote change.